pkg/resources/management.cattle.io/v3/projectroletemplatebinding/ProjectRoleTemplateBinding.md

```diff
@@ -9,10 +9,12 @@ This is to prevent privilege escalation.
 
 Users cannot create ProjectRoleTemplateBindings that violate the following constraints:
 
+- The projectName field must have the format `cluster-id:project-id`, and `project-id` must match the namespace of the
```

Comment:
This is talking about the same `ProjectName` as mentioned on line 17 right? Maybe state both requirements ("must be specified" and "have the format...") on the same line.

Reply:
Adjusted!